verify.sh

```diff
@@ -1,3 +1,9 @@
 #!/bin/bash
-flake8 rabbitmq.py test_rabbitmq.py metric_info.py
+flake8 rabbitmq.py metric_info.py
```

Comment:
It is simpler to just put `set -e` at the top of the script so that anything that returns non-zero will cause the script to exit with that same non-zero exit code.